spec/factories/season.rb

```diff
@@ -1,9 +1,5 @@
 FactoryBot.define do
   factory :season do
-    sequence(:name) { |n| "#{Date.today.year}-#{n}" }
-    starts_at { Time.utc(Date.today.year, 7, 1) }
-    ends_at { Time.utc(Date.today.year, 9, 30) }
-    applications_open_at { Time.utc(Date.today.year, 3, 1) }
-    applications_close_at { Time.utc(Date.today.year, 3, 31) }
+    sequence(:name, '2000')
```

Comment:
Ist that a sequence? There‘s no iterator var being evaluated

Reply:
That's because it's a even simpler sequence 😉 : starts at 2000 and just increments by 1.

Reply:
Interesting, So it `to_i`s it under the hood? Neat.

Reply:
Way simpler. It just calls `next` ✌️ 
Try it yourself:
```ruby
[1] pry(main)> "2015".next
=> "2016"
```
😹  Ruby I guess 😹

Reply:
Whoa! TIL about `String#(next|succ)`
![](https://images.duckduckgo.com/iu/?u=https%3A%2F%2Fircimg.net%2Fmind-blown.gif&f=1)

Reply:
(on 2nd thought, this is ofc how `"2000".."2050"` works in the first place. D'oh.